test/int/deployment_test.go

```diff
@@ -1329,3 +1338,74 @@ func float64ptr(f float64) *float64 {
 func boolptr(b bool) *bool {
 	return &b
 }
+
+func createConnectionSecret() *corev1.Secret {
+	var connectionSecret corev1.Secret
+	By(fmt.Sprintf("Creating the Secret %s", kube.ObjectKeyFromObject(&connectionSecret)), func() {
```

Comment:
Won't this display an empty name and a namespace?

Reply:
Yep, seems I made a mistake here trying to display the secret data before it was prepared.

Reply:
fixed